pkgs/development/compilers/rust/1_67.nix

```diff
@@ -66,6 +66,12 @@ import ./default.nix {
       url = "https://github.com/rust-lang/rust/commit/de363d54c40a378717881240e719f5f7223ba376.patch";
       hash = "sha256-3Xb803LZUZ1dldxGJ65Iw6gg1V1K827OB/0b32GqilU=";
     })
+    (fetchpatch {
+      name = "re-erased-regions-are-local.patch";
+      url = "https://github.com/rust-lang/rust/commit/9d110847ab7f6aef56a8cd20cb6cea4fbcc51cd9.patch";
+      excludes = [ "*tests/*" ];
```

Comment:
Should I mention why this is done? (Tests aren't included in rustc source tarballs.)

Reply:
github didn't give me a notification for some reason, I think it would be nice to mention that. I usually also link to the pull request, but it's up to you

Reply:
You didn't get a notification because it's a draft :)

Reply:
Added.